pages/common/salt.md

```diff
@@ -1,6 +1,7 @@
 # salt
 
 > Execute commands and assert state on remote salt minions.
+> This command also has documentation about its subcommands, e.g. `salt run`.
```

Comment:
Looks like all the `salt-*` commands in tldr are not subcommands from `salt`.

Reply:
Like with `ssh`, `salt-run` and the others are official, so I think we can keep them nonetheless.